src/Hidden/Hidden.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+/**
+ * Responsively hides children based on the selected implementation.
```

Comment:
Let's try to be consistent with the codebase, I think that we put that comment right above the component definition.